sql-plugin/src/main/311until330-all/scala/com/nvidia/spark/rapids/shims/v2/GpuParquetScan.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2021, NVIDIA CORPORATION.
+ * Copyright (c) 2020-2022, NVIDIA CORPORATION.
```

Comment:
Updating the copyright on a file that is otherwise untouched.

Reply:
Fixed